src/locale/en.json

```diff
@@ -1,5 +1,10 @@
 {
-	"home.page.title": "Content management - Civil Service Learning",
+	"content_management_title": "Content management",
+	"add_course_title_title": "Add course title",
+	"add_course_details_title": "Add course title",
```

Comment:
INFO - same translation for two different key values?

Reply:
Fixed!